diff --git a/packstack/puppet/templates/amqp.pp b/packstack/puppet/templates/amqp.pp
index 31c65d5..4150ee4 100644
--- a/packstack/puppet/templates/amqp.pp
+++ b/packstack/puppet/templates/amqp.pp
@@ -34,12 +34,30 @@ define enable_rabbitmq {
 }
 
 define enable_qpid($enable_ssl = 'n', $enable_auth = 'n') {
+  case $::operatingsystem {
+    'Fedora': {
+      if (is_integer($::operatingsystemrelease) and $::operatingsystemrelease >= 20) or $::operatingsystemrelease == "Rawhide" {
+        $config = '/etc/qpid/qpidd.conf'
+      } else {
+        $config = '/etc/qpidd.conf'
+      }
+    }
+
+    'RedHat': {
+      if $::operatingsystemmajrelease >= 7 {
+        $config = '/etc/qpid/qpidd.conf'
+      } else {
+        $config = '/etc/qpidd.conf'
+      }
+    }
+
+    default: {
+      $config = '/etc/qpidd.conf'
+    }
+  }
 
   class {"qpid::server":
-	  config_file => $::operatingsystem? {
-	    'Fedora' => '/etc/qpid/qpidd.conf',
-	    default  => '/etc/qpidd.conf',
-	    },
+	  config_file => $config,
     auth => $enable_auth ? {
 	    'y'  => 'yes',
 	    default => 'no',
